tests/test_decorators.py

```diff
@@ -60,3 +61,28 @@ def test_mutation_refresh_jwt_token_required(flask_app: Flask):
         assert get_jwt_data(response['refresh']["newToken"], "access")["identity"] == "username"
 
 
+def test_extract_token_value_from_params(flask_app):
+    token_name = "token"
+    token_value = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9"
+    request_params = {token_name: token_value}
+    assert _extract_token_value(request_params, token_name) == token_value
+
```

Comment:
Please add decorator tests

Reply:
I can't seem to be able to construct a query that works with the existing schema without the protected part.

Can you show me an example?

Reply:
Ok, I'll modify schema and add tests after merging this.